portfolio/src/main/webapp/script.js

```diff
@@ -15,7 +15,7 @@
 /**
  * Adds a random fact to the page.
  */
-function addRandomFact() {
+/* function addRandomFact() {
```

Comment:
Instead of commenting it out. I'd recommend deleting it. We can always grab it from the history if you want to add it back!